src/Model/Command/UserMerge.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\Model\Command;
+
+/**
+ * Merge two users. DELETES source user!
+ */
+class UserMerge extends AbstractCommand
+{
+    /** @var string */
+    private $sourceUserId;
+    /** @var string */
+    private $targetUserId;
+
+    /**
+     * Merge source user to target user AND DELETE SOURCE USER.
+     */
+    public function __construct(string $sourceUserId, string $targetUserId)
+    {
+        $this->sourceUserId = $sourceUserId;
+        $this->targetUserId = $targetUserId;
+    }
+
+    public function getCommandType(): string
+    {
+        return 'user-merge';
+    }
+
+    public function getCommandParameters(): array
+    {
+        return [
+            'target_user_id' => $this->targetUserId,
+            'source_user_id' => $this->sourceUserId,
+        ];
+    }
+
+    /**
+     * Merge source user to target user AND DELETE SOURCE USER.
+     */
+    public static function create(string $sourceUserId, string $targetUserId): self
+    {
+        return new static($sourceUserId, $targetUserId);
+    }
+
+    /**
+     * Merge source user into target user AND DELETE SOURCE USER.
+     */
+    public static function mergeInto(string $targetUserId, string $sourceUserId): self
```

Comment:
Maybe make the "delete" thing more obvious by naming the variable eg. `$sourceUserIdToBeDeleted` :)?